### PR TITLE
Added recipe for pytest-monitor, release 1.0.0

### PR DESCRIPTION
**Pull request by js-dieu**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by js-dieu** [comment]
I am willing to maintain the recipe (official maintainer of pytest-monitor)

**Comment by marcelotrevisani** [comment]
Could you please revert the files which you removed?

**Comment by js-dieu** [comment]
@marcelotrevisani Sure, will do.

File: recipes/pytest-monitor/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  number: 0
+  skip: True  # [py<35 or win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - memory_profiler
+    - pytest
+    - psutil
+    - requests
```
Comment (scopatz) [inline recipes/pytest-monitor/meta.yaml]: I don't think these are needed in build (or at install time). They are not compilers

Comment (js-dieu) [reply recipes/pytest-monitor/meta.yaml]: Thanks for spotting. I just got completely confused.

File: recipes/pytest-monitor/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/CFMTech/pytest-monitor/archive/{{ name }}-{{ version }}.tar.gz
+  sha256: b4081347adcbc13c23eac3384a3dfa83a1c6301bb3aaae90e5588ec30a231ecd
+
+build:
+  number: 0
```
Comment (xhochy) [inline recipes/pytest-monitor/meta.yaml]: This can be noarch as it doesn't have any native code.
```suggestion
  number: 0
  noarch: python
```

Comment (js-dieu) [reply recipes/pytest-monitor/meta.yaml]: Hello

Thanks for your feedback.
I don't support windows and I am not willing to. This is the reason why I put the build selector. The point with this approach is that it forbids me to have noarch packages.

Comment (xhochy) [reply recipes/pytest-monitor/meta.yaml]: Building packages separate for all Python version even if it isn't needed wastes CI resources and introduces a maintenance burden. Feel free to close issues in the package issue tracker about this but I don't think we should put this burden artifically on the conda-forge shoulders.

Comment (js-dieu) [reply recipes/pytest-monitor/meta.yaml]: @xhochy I understand your point. I'll see what I can do to make it pass. Thanks for your review.

File: recipes/pytest-monitor/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  noarch: python
+  script: "{{ PYTHON }} -m pip install --no-deps  . -vv"
+
+requirements:
+  host:
+    - python
```
Comment (xhochy) [inline recipes/pytest-monitor/meta.yaml]: If you want to limit the Python version, you can use in `host` and `run` the following:
```suggestion
    - python >=3.6
```

Comment (js-dieu) [reply recipes/pytest-monitor/meta.yaml]: Noted, thanks!

File: recipes/pytest-monitor/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,48 @@
+{% set name = "pytest-monitor" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: //https://pypi.io/packages/source/p/{{ name }}/{{ name }}-{{ version }}.tar.gz
```
Comment (xhochy) [inline recipes/pytest-monitor/meta.yaml]: Seems like there are two too much slashes
```suggestion
  url: https://pypi.io/packages/source/p/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Comment (js-dieu) [reply recipes/pytest-monitor/meta.yaml]: message crossing :) Thanks !

**Comment by js-dieu** [comment]
@xhochy Any idea as to why the ci/appveyor thing constantly fails with such message? Do I need to do something special ?

**Comment by js-dieu** [comment]
Thanks @xhochy  for you reactivity!

**Pull request merged**